test/acceptance/accessibility.py

```diff
@@ -243,6 +243,78 @@ def test_multiple_rubrics(self):
         self._check_a11y(self.staff_area_page)
 
 
+class MultipleOpenAssessmentA11yTest(OpenAssessmentA11yTest, FullWorkflowMixin):
+    """
+    Test accessibility when we have a unit containing multiple ORA blocks.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentA11yTest, self).setUp('multiple_ora', staff=True)
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    # TODO: remove this method override. See TNL-1593
+    def _check_a11y(self, page):
+        page.a11y_audit.config.set_scope(
+            exclude=[
+                ".container-footer",
+                ".nav-skip",
+                "#global-navigation",
+                "#openassessment",
+                "#openassessment__message"
+                "#openassessment__steps"
+                "#openassessment__response"
+                "#openassessment__student-training"
+                "#openassessment__peer-assessment"
+                "#openassessment__staff-assessment"
+                "#openassessment__grade"
+            ],
+        )
+        page.a11y_audit.config.set_rules({
+            "ignore": [
+                "color-contrast",  # TODO: AC-198
+                "empty-heading",  # TODO: AC-197
+                "link-href",  # TODO: AC-199
+                "link-name",  # TODO: AC-196
+                "skip-link",  # TODO: AC-179
+            ]
+        })
+        page.a11y_audit.check_for_accessibility_errors()
+
+    def check_a11y(self):
+        """ check accessibility"""
+        self._check_a11y(self.submission_page)
```

Comment:
You don't actually need to call _check_a11y on all these different things. One call suffices, because the check does the whole HTML page at once. So this test point can be boiled down to just (note that it is important to actually visit the page you are testing):

````
self.submission_page.visit().submit_response(self.SUBMISSION)
self._check_a11y(self.submission_page)
````

A better test though would be going through the workflow and doing the accessibility check at different points, as different parts of the UI appear (take a look at the existing accessibility tests as an example).